src/main/java/org/jitsi/jigasi/TranscriptionGateway.java

```diff
@@ -106,15 +106,17 @@ private String getCustomTranscriptionServiceClass(String tenant)
                         REMOTE_TRANSCRIPTION_CONFIG_URL,
                         null);
 
-        if (remoteTranscriptionConfigUrl != null)
+        if (remoteTranscriptionConfigUrl != null && tenant != null)
         {
+            logger.info("Retrieving transcriber from remote URL " + remoteTranscriptionConfigUrl + "/" + tenant);
```

Comment:
Do we need that log on every conference?

Reply:
moved to debug